### PR TITLE
Send and receive tasks

### PR DESCRIPTION
**Pull request by btry**
Flyve MDM plugin for GLPI creates tasks in this plugin. Automatic actions in this plugin are used to send the task to the server and poll until they finish (successfully or not).

Note that this can be enhanced thanks to the callback feature of Orion. This requires a service account in GLPI created by this plugin.

Note also that the size of applications we can send is limited because the current implementation reads the whole file, base64 encodes it to satisfy the remote API service.

### CODE REVIEW

**Comment by ajsb85** [comment]
Hi, @btry 
Use the commit type "ci" instead "chore" for the webhook notification

**Comment by btry** [comment]
done

File: .travis.yml [hunk 1]
```diff
@@ -42,3 +42,4 @@ cache:
     
 notifications:
   webhooks: https://fathomless-fjord-24024.herokuapp.com/notify
```
Comment (ajsb85) [inline .travis.yml]: You can safely remove this webhook URL

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Comment by ajsb85** [comment]
I change the title to [WIP] because the Travis CI build failed.

**Pull request merged**